src/components/LogPane/index.tsx

```diff
@@ -11,40 +13,42 @@ interface LogPaneProps {
   rowCount: number;
 }
 const LogPane: React.FC<LogPaneProps> = ({ rowCount, rowRenderer }) => {
-  const { listRef, processedLogLines, scrollToLine } = useLogContext();
+  const { listRef, preferences, processedLogLines, scrollToLine } =
+    useLogContext();
+  const { setWrap } = preferences;
 
   const [shareLine] = useQueryParam<number | undefined>(
     QueryParams.ShareLine,
     undefined
   );
 
   useEffect(() => {
-    const initialScrollIndex = findLineIndex(processedLogLines, shareLine);
-    if (initialScrollIndex > -1) {
-      leaveBreadcrumb(
-        "Triggered scroll to shareLine",
-        {
-          initialScrollIndex,
-          shareLine,
-        },
-        "user"
-      );
-      // This timeout is necessary to ensure that the list has been rendered
-      // before we try to scroll to the line.
+    if (listRef.current) {
+      // Use a timeout to execute certain actions after the log pane has rendered. All of the
+      // code below describes one-time events.
       setTimeout(() => {
-        scrollToLine(initialScrollIndex);
+        const initialScrollIndex = findLineIndex(processedLogLines, shareLine);
+        if (initialScrollIndex > -1) {
+          leaveBreadcrumb(
+            "Triggered scroll to shareLine",
+            { initialScrollIndex, shareLine },
+            "user"
+          );
+          scrollToLine(initialScrollIndex);
+        } else {
+          leaveBreadcrumb(
+            "shareLine not provided or found in processedLogLines",
+            { shareLine },
+            "process"
+          );
+        }
+
+        // Wrap can be enabled after the log pane has initially loaded.
+        setWrap(Cookies.get(WRAP) === "true");
```

Comment:
Should we only call this if necessary?
```suggestion
        if (Cookies.get(WRAP) === "true") {
          setWrap(true);
        }
```